apps/api/src/billing/routes/stripe-transactions/stripe-transactions.router.ts

```
@@ -47,12 +51,31 @@ const confirmPaymentRoute = createRoute({
+    userId: data.userId,
+    paymentMethodId: data.paymentMethodId,
+    amount: data.amount,
+    currency: data.currency
+  });
+
+  // Check if 3D Secure is required
+  if (result.data.requiresAction) {
+    return c.json(result, 202);
+  }
+
+  return c.json(result, 200);
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)